src/Form/DataTransformer/ModelToIdPropertyTransformer.php

```diff
@@ -170,20 +169,14 @@ public function transform($value)
             }
         }
 
-        if (empty($this->property)) {
+        if ('' === $this->property) {
```

Comment:
If needed, this check seems better in the constructor.

Reply:
I would prefer not to change behavior in this case, I think with empty string, things are more clear than using empty, but I can revert it if you prefer, just tell me

Reply:
I would prefer to not change behavior if it is not needed on this PR, I think it is clearer comparing with '' instead of calling empty function, but if you prefer, I can revert it.